test/scripts/e2e/test_kfservice_creating.py

```diff
@@ -0,0 +1,19 @@
+import time
```

Comment:
Can we put these tests in tests/e2e/...

I expect in the future we will have tests/performance, /tests/common, etc

Reply:
Great comments!